src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_help.py

```diff
@@ -0,0 +1,206 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['bot'] = """
+    type: group
+    short-summary: Manage Microsoft Bot Services.
+"""
+helps['bot create'] = """
+    type: command
+    short-summary: Create a new Microsoft Bot Service.
+"""
+helps['bot show'] = """
+    type: command
+    short-summary: Get an existing Microsoft Bot Service.
+"""
+helps['bot prepare-publish'] = """
+    type: command
+    short-summary: Add scripts to your local source code directory to
+                   be able to publish back using `az bot publish`.
+"""
+helps['bot delete'] = """
+    type: command
+    short-summary: Delete an existing Bot Service.
+"""
+helps['bot update'] = """
+    type: command
+    short-summary: Update an existing Microsoft Bot Service.
+    examples:
+        - name: Update description on a bot
+          text: |-
+            az bot update -n botName -g MyResourceGroup --set properties.description="some description"
+"""
+helps['bot publish'] = """
+    type: command
+    short-summary: Publish to an existing Microsoft Bot Service.
+    long-summary: Publish your source code to your Azure Web App.
```

Comment:
The `short-summary` and `long-summary` here read as contradictory. It implies this is published to two different services.

Reply:
the publish is actually done to an app service associated with the bot. so made that clear now.